usaspending_api/search/migrations/0010_subaward_search.py

```diff
@@ -0,0 +1,487 @@
+                ('business_categories', django.contrib.postgres.fields.ArrayField(base_field=models.TextField(), default=list, size=None)),
+                ('treasury_account_identifiers', django.contrib.postgres.fields.ArrayField(base_field=models.IntegerField(), default=None, size=None)),
+                ('pulled_from', models.TextField(blank=True, null=True)),
+                ('type_of_contract_pricing', models.TextField(blank=True, null=True)),
+                ('type_set_aside', models.TextField(blank=True, null=True)),
+                ('extent_competed', models.TextField(blank=True, null=True)),
+                ('product_or_service_code', models.TextField(blank=True, null=True)),
+                ('product_or_service_description', models.TextField(blank=True, null=True)),
+                ('sub_legal_entity_country_code', models.TextField(blank=True, null=True)),
+                ('sub_legal_entity_country_name', models.TextField(blank=True, null=True)),
+                ('sub_legal_entity_county_code', models.TextField(blank=True, null=True)),
+                ('sub_legal_entity_county_name', models.TextField(blank=True, null=True)),
+                ('sub_legal_entity_zip5', models.TextField(blank=True, null=True)),
+                ('sub_legal_entity_city_code', models.TextField(blank=True, null=True)),
+                ('sub_legal_entity_congressional', models.TextField(blank=True, null=True)),
+                ('place_of_perform_scope', models.TextField(blank=True, null=True)),
+                ('sub_place_of_perform_country_co', models.TextField(blank=True, null=True)),
+                ('sub_place_of_perform_country_name', models.TextField(blank=True, null=True)),
+                ('sub_place_of_perform_county_code', models.TextField(blank=True, null=True)),
+                ('sub_place_of_perform_county_name', models.TextField(blank=True, null=True)),
+                ('sub_place_of_perform_zip5', models.TextField(blank=True, null=True)),
+                ('sub_place_of_perform_city_code', models.TextField(blank=True, null=True)),
+                ('sub_place_of_perform_congressio', models.TextField(blank=True, null=True)),
+                ('keyword_ts_vector', django.contrib.postgres.search.SearchVectorField()),
+                ('award_ts_vector', django.contrib.postgres.search.SearchVectorField()),
+                ('recipient_name_ts_vector', django.contrib.postgres.search.SearchVectorField()),
+                ('award', models.BigIntegerField(null=True, db_column="award_id")),
+                ('awarding_agency', models.IntegerField(null=True, db_column="awarding_agency_id")),
+                ('cfda', models.IntegerField(null=True, db_column="cfda_id")),
+                ('funding_agency', models.IntegerField(null=True, db_column="funding_agency_id")),
+            ],
+            options={
+                'db_table': 'rpt"."subaward_search',
```

Comment:
Is this a typo, or am I just having a hard time parsing ...?
`'db_table': 'rpt"."subaward_search'`